itests/kit/ensemble.go

```diff
@@ -335,7 +336,49 @@ func (n *Ensemble) Start() *Ensemble {
 
 	// Create all inactive full nodes.
 	for i, full := range n.inactive.fullnodes {
-		r := repo.NewMemory(nil)
+
+		var r repo.Repo
+		if !full.options.fsrepo {
+			rmem := repo.NewMemory(nil)
+			n.t.Cleanup(rmem.Cleanup)
+			r = rmem
+		} else {
+			repoPath, err := ioutil.TempDir(os.TempDir(), "lotus-itest-fsrepo-temp-")
```

Comment:
Coldstore prune won't work unless we have a badger BS so I'm introducing this.  Please double check I'm not leaving any stray testing files behind